ci/README.md

```diff
@@ -32,11 +32,13 @@ Questionnaire state can be backed up using the `backup_questionnaire_state.yaml`
 ```sh
 PROJECT_ID=<project_id> \
 BUCKET_NAME=<bucket_name> \
+FILE_PREFIX=<file_prefix> \
```

Comment:
Not sure about this name. If I read the docs it's a folder not a file prefix and it's only used for backups. Wouldn't BACKUP_FOLDER_NAME or similar be more appropriate?

Reply:
The card says BACKUP_PREFIX which makes me even more confused